text/0006-shipyard-next-generation.md

```diff
@@ -0,0 +1,317 @@
+# The next generation of Shipyard
+
+The next generation of Shipyard to allow custom delivery/remediation processes.
+
+## Motivation
+
+*There's always room for smart enhancements* - the Shipyard specification is no exception.
+
+A shipyard allows a **Site Reliability Engineer (SRE)** to specify WHAT should happen in the course of an artifact delivery or remediation process. Due to certain use cases, it has been identified that the current specification of the shipyard has limitations and hides details from the SRE. This KEP proposes an enhancement of the *Shipyard* specification to bring flexibility into the definition of the *WHAT*, but still provides an opinionated approach and smart defaults.
+
+Please read this KEP in combination with the KEP: [First version of a Keptn uniform](https://github.com/keptn/enhancement-proposals/pull/7); especially regarding the eventing part.
+
+## Explanation
+
+:man:/:blonde_woman: The target persona of this KEP is an *SRE*, who is responsible for processes and workflows to automate application delivery or remediation operations. 
+
+Before getting started, a brief recap of the important terms and definitions used in this KEP:
+
+- **Project stage:** A project stage (or just stage) defines a logical space in an environment, which has a dedicated purpose for an application in a continuous delivery process. The project stages are ordered, meaning that a stage has at least a previous and/or next stage. 
```

Comment:
> meaning that a stage has at least a previous and/or next stage. 

This is not entirely true (e.g., single-stage projects). 

> The project stages are ordered

This also doesn't hold with the new shipyard format. Stages are not ordered, they can be triggered in any order (or even in parallel).

Reply:
I gave it a try to improve this paragraph.